text/0014-improve-locking-mechanism-for-parachains.md

```diff
@@ -0,0 +1,105 @@
+# RFC-0014: Improve locking mechanism for parachains
+
+|                 |                                          |
+| --------------- | ---------------------------------------- |
+| **Start Date**  | July 25, 2023                            |
+| **Description** | Improve locking mechanism for parachains |
+| **Authors**     | Bryan Chen                               |
+
+## Summary
+
+The manager of a parachain have the permission to manage the parachain when the parachain is unlocked. Parachain is by default locked when onboarded on a slot. This requires the parachain wasm/genesis must be valid otherwise a root track governance action on relaychain is required to update the parachain.
```

Comment:
```suggestion
The manager of a parachain has permission to manage the parachain when the parachain is unlocked. Parachains are by default locked when onboarded to a slot. This requires the parachain wasm/genesis must be valid, otherwise a root track governance action on relaychain is required to update the parachain.
```

Reply:
This summary is 99% about the problem, not about the solution. It should be more about the solution.